openid_connect.module

```diff
@@ -1387,15 +1345,15 @@ function openid_connect_complete_authorization($client, $tokens, $destination) {
       WATCHDOG_DEBUG
     );
   }
-  if (isset($userinfo['state'])) {
+  if (isset($userinfo['address']['region'])) {
```

Comment:
this looks like correct approach, not sure why we did not use it before - you did test that with this syntax we are getting correct state? I guess we can only test after we push this code to remote instance